sympy/physics/continuum_mechanics/truss.py

```diff
@@ -0,0 +1,598 @@
+        >>> t.add_node('C', 2, 2)
+        >>> t.add_member('AB', 'A', 'B')
+        >>> t.add_member('AC', 'A', 'C')
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        [['AB', 'A', 'B'], ['AC', 'A', 'C'], ['BC', 'B', 'C']]
+        >>> t.remove_member('AC')
+        >>> t.members
+        [['AB', 'A', 'B'], ['BC', 'B', 'C']]
+        """
+        if label not in self._member_labels:
+            raise ValueError("No such member exists in the Truss")
+
+        else:
+            self._members.remove([label, self._member_nodes[label][0], self._member_nodes[label][1]])
+            self._member_labels.remove(label)
+            self._nodes_occupied.pop(tuple([self._member_nodes[label][0], self._member_nodes[label][1]]))
+            self._nodes_occupied.pop(tuple([self._member_nodes[label][1], self._member_nodes[label][0]]))
+            self._member_nodes.pop(label)
+            self._internal_forces.pop(label)
+
+    def change_node_label(self, label, new_label):
+        """
+        This method changes the label of a node.
+
+        Parameters
+        ==========
+        label: String or Symbol
+            The label of the node for which the label has
+            to be changed.
+
+        new_label: String or Symbol
+            The new label of the node.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.nodes
+        [('A', 0, 0), ('B', 3, 0)]
+        >>> t.change_node_label('A', 'C')
+        >>> t.nodes
+        [('C', 0, 0), ('B', 3, 0)]
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        [['BC', 'B', 'C']]
+        >>> t.change_member_label('BC', 'BC_new')
+        >>> t.members
+        [['BC_new', 'B', 'C']]
+        """
+        if label not in self._node_labels:
+            raise ValueError("No such node exists for the Truss")
+        else:
+            for node in self._nodes:
+                if node[0] == label:
+                    if self._supports[label] == 'pinned':
+                        if 'R_'+str(label)+'_x' in list(self._reaction_loads) and 'R_'+str(label)+'_y' in list(self._reaction_loads):
+                            self._reaction_loads['R_'+str(new_label)+'_x'] = self._reaction_loads['R_'+str(label)+'_x']
+                            self._reaction_loads['R_'+str(new_label)+'_y'] = self._reaction_loads['R_'+str(label)+'_y']
+                            self._reaction_loads.pop('R_'+str(label)+'_x')
+                            self._reaction_loads.pop('R_'+str(label)+'_y')
+                        self._loads['R_'+str(new_label)+'_x'] = self._loads['R_'+str(label)+'_x']
+                        self._loads['R_'+str(new_label)+'_y'] = self._loads['R_'+str(label)+'_y']
+                        self._loads.pop('R_'+str(label)+'_x')
+                        self._loads.pop('R_'+str(label)+'_y')
```

Comment:
These lines throw an error when `change_node_label` is called  after applying support to a node. I think this is happening because the `self._loads` has node labels as keys, not forces. So this should be something like
`self._loads[new label]['R_'+str(new_label)+'_x'] = ......`